src/Symfony/Component/Validator/Constraints/PasswordStrengthValidator.php

```diff
@@ -15,15 +15,26 @@
 use Symfony\Component\Validator\ConstraintValidator;
 use Symfony\Component\Validator\Exception\UnexpectedTypeException;
 use Symfony\Component\Validator\Exception\UnexpectedValueException;
+use Symfony\Component\Validator\Password\PasswordStrengthEstimatorInterface;
 
 final class PasswordStrengthValidator extends ConstraintValidator
 {
-    /**
-     * @param (\Closure(string):PasswordStrength::STRENGTH_*)|null $passwordStrengthEstimator
-     */
+    private \Closure $passwordStrengthEstimator;
+
     public function __construct(
-        private readonly ?\Closure $passwordStrengthEstimator = null,
+        \Closure|PasswordStrengthEstimatorInterface $passwordStrengthEstimator,
```

Comment:
removing the default value is a BC break as passing an argument to the constructor is now mandatory

Reply:
the PR also changes the FrameworkBundle configuration to pass by default the new Class so that covers the removing the null

Reply:
The Validator component can be used in a standalone way without the FrameworkBundle where developers instantiate the `PasswordStrengthValidator` themselves.

Reply:
When the validator is used standalone and without a service container, validators are instantiated with a parameterless constructor call. After your change, this won't be possible anymore. Given that we have a good default implementation for a strength estimator, breaking that workflow feels unnecessary.

```suggestion
        \Closure|PasswordStrengthEstimatorInterface|null $passwordStrengthEstimator = null,
```